Orange/data/io.py

```diff
@@ -29,6 +29,9 @@
 from Orange.util import Registry, flatten, namegen
 
 
+__all__ = ["Flags"]
```

Comment:
Added `FileFormat`. As said, the individual readers are really just current implementations that should not be used directly elsewhere, so I don't think they belong in `__all__`. And they can be imported from data.io if needed (for exceptional cases only).